docs/UsersGuide/source/RunSRW.rst

```diff
@@ -629,38 +615,44 @@ In the ``config.yaml`` file, set ``MACHINE: macos`` or ``MACHINE: linux``, and m
       PREEXISTING_DIR_METHOD: rename
       VERBOSE: true
       COMPILER: gnu
+   task_get_extrn_ics:
+      USE_USER_STAGED_EXTRN_FILES: true
+      EXTRN_MDL_SOURCE_BASEDIR_ICS: /path/to/input_model_data/FV3GFS/grib2/2019061518
+      EXTRN_MDL_DATA_STORES: disk
+   task_get_extrn_lbcs:
+      USE_USER_STAGED_EXTRN_FILES: true
+      EXTRN_MDL_SOURCE_BASEDIR_LBCS: /path/to/input_model_data/FV3GFS/grib2/2019061518
+      EXTRN_MDL_DATA_STORES: disk
    task_run_fcst:
       PREDEF_GRID_NAME: RRFS_CONUS_25km	
       QUILTING: true
 
-Due to the limited number of processors on MacOS systems, users must also configure the domain decomposition defaults (usually, there are only 8 CPUs in M1-family chips and 4 CPUs for x86_64 chips). 
+Due to the limited number of processors on MacOS systems, users must also configure the domain decomposition parameters directly in section of the ``predef_grid_params.yaml`` file pertaining to the grid they want to use. Domain decomposition needs to take into the account number of available CPUs and configure variables ``LAYOUT_X``, ``LAYOUT_Y``, and ``WRTCMP_write_tasks_per_group``. 
```

Comment:
Rather than "take into the account number", should this read "take into account the number"?  Additionally, should "accordingly" be added to the end of sentence (i.e., configure variables ``LAYOUT_X``, ``LAYOUT_Y``, and ``WRTCMP_write_tasks_per_group``, accordingly)?